tests/E2E/RetryPayment/RetryingPaymentTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace E2E\RetryPayment;
+
+use Tests\CommerceWeavers\SyliusTpayPlugin\Api\Utils\OrderPlacerTrait;
+use Tests\CommerceWeavers\SyliusTpayPlugin\E2E\E2ETestCase;
+use Tests\CommerceWeavers\SyliusTpayPlugin\E2E\Helper\Order\RetryPaymentTrait;
+
+final class RetryingPaymentTest extends E2ETestCase
+{
+    use OrderPlacerTrait;
```

Comment:
`OrderExampleFactory` has been making troubles; I've reused order placer from the API and it works xD